.devcontainer/qiskit/devcontainer.json

```diff
@@ -1,7 +1,7 @@
 {
   "image": "mcr.microsoft.com/devcontainers/python:3.12-bullseye",
   "waitFor": "onCreateCommand",
-  "updateContentCommand": "python3 -m pip install -r Requirements/qiskit.txt",
+  "updateContentCommand": "/usr/bin/python3 -m pip install -r Requirements/qiskit.txt && /usr/local/bin/python3 -m pip install -r Requirements/qiskit.txt",
```

Comment:
Specified location for the installations, and added a second part which included installation to local/bin/ in order to allow the use of python 3.12 interpreters, as that is the corresponding location for them.